src/AdyenNativeModules.js

```diff
@@ -50,6 +50,8 @@ export function getNativeComponent(name, paymentMethods) {
     case 'apple-pay':
       return { nativeComponent: AdyenApplePay };
     case 'paywithgoogle':
+    case 'googlepay':
+    case 'google-pay':
```

Comment:
Why do we need `google-pay`? I see we did the same for apple pay.

Reply:
Not really needed. I'll remove it 🤔

Reply:
fixed [14cfd19](https://github.com/Adyen/adyen-react-native/pull/105/commits/14cfd19033353fdc4fea058ac90564e7e5433d4b)